index.html

```diff
@@ -0,0 +1,114 @@
+        <div>
+          <b>1935</b> - Has to stop school and save up more money. Works in the
+          Civilian Conservation Corps, helping starving Americans. "I saw how
+          food changed them", he said. "All of this left scars on me."
+        </div>
+        <div>
+          <b>1937</b>- Finishes university and takes a job in the US Forestry
+          Service
+        </div>
+        <div>
+          <b>1938</b> - Marries wife of 69 years Margret Gibson. Gets laid off
+          due to budget cuts. Inspired by Elvin Charles Stakman, he returns to
+          school study under Stakman, who teaches him about breeding
+          pest-resistent plants
+        </div>
+        <div>
+          <b>1941</b>- Tries to enroll in the military after the Pearl Harbor
+          attack, but is rejected. Instead, the military asked his lab to work
+          on waterproof glue, DDT to control malaria, disinfectants, and other
+          applied science.
+        </div>
+        <div><b>1942</b> - Receives a Ph.D. in Genetics and Plant Pathology</div>
+        <div>
+          <b>1944</b>- Rejects a 100% salary increase from Dupont, leaves behind
+          his pregnant wife, and flies to Mexico to head a new plant pathology
+          program. Over the next 16 years, his team breeds 6,000 different
+          strains of disease resistent wheat - including different varieties for
+          each major climate on Earth.
+        </div>
+        <div>
+          <b>1945</b>- Discovers a way to grown wheat twice each season,
+          doubling wheat yields
+        </div>
+        <div>
+          <b>1953</b>- crosses a short, sturdy dwarf breed of wheat with a
+          high-yeidling American breed, creating a strain that responds well to
+          fertilizer. It goes on to provide 95% of Mexico's wheat.
+        </div>
+        <div>
+          <b>1962</b> - Visits Delhi and brings his high-yielding strains of
+          wheat to the Indian subcontinent in time to help mitigate mass
+          starvation due to a rapidly expanding population
+        </div>
+        <div><b>1970</b>- receives the Nobel Peace Prize</div>
+        <div>
+          <b>1983</b>- helps seven African countries dramatically increase their
+          maize and sorghum yields
+        </div>
+        <div>
+          <b>1984</b> - becomes a distinguished professor at Texas A&M
+          University
+        </div>
+        <div>
+          <b>2005</b>- states "we will have to double the world food supply by
+          2050." Argues that genetically modified crops are the only way we can
+          meet the demand, as we run out of arable land. Says that GM crops are
+          not inherently dangerous because "we've been genetically modifying
+          plants and animals for a long time. Long before we called it science,
+          people were selecting the best breeds."
+        </div>
+        <div><b>2009</b>- dies at the age of 95.</div>
+      </div>
```

Comment:
why did you use `div` tags to build a list instead of semantic HTML `ul` and `li`

Reply:
- [ ] I recommend following my previous comments because they will help in improving your communication skills :100:.